bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/ValueCreator.java

```diff
@@ -680,22 +682,47 @@ public static BXml createXmlProcessingInstruction(BString target, BString data,
      *
      * @param mapType map type.
      * @return map value
+     * @deprecated use {@link #createMapValue(MapType)} instead
      */
+    @Deprecated
```

Comment:
Shall we create an issue in th https://github.com/ballerina-platform/ballerina-standard-library repo mentioning the API deprecation? Better if we can mention the modules that need the change. 
Similar to https://github.com/ballerina-platform/ballerina-standard-library/issues/1646

Reply:
Created https://github.com/ballerina-platform/ballerina-standard-library/issues/2568.